packages/vx-demo/src/components/tiles/Drag-i.tsx

```diff
@@ -40,8 +47,11 @@ const generateItems = ({ width, height }: WidthAndHeight) =>
   });
 
 export default function DragI({ width, height }: ShowProvidedProps) {
-  const memoizedItems = useMemo(() => generateItems({ width, height }), [width, height]);
```

Comment:
this was generating items when width/height was ~0, and `useState` doesn't update if the default updates (when this memoized value was recomputed)